web/client/utils/cesium/__tests__/WMSUtils-test.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+*/
+import expect from 'expect';
+import { wmsToCesiumOptionsBIL } from '../WMSUtils';
+
+const testLayerConfig = {
+    "type": "terrain",
+    "provider": "wms",
```

Comment:
Please use the wms config with useForElevantion here, the type terrain is not available on this branch